grammar.js

```diff
@@ -748,6 +751,6 @@ function commaSep1(rule) {
   return seq(rule, repeat(seq(',', rule)))
 }
 
-function sepTrailing (separator, recurSymbol, rule) {
+function sepTrailing (separator, rule, recurSymbol) {
```

Comment:
@maxbrunsfeld this change is unrelated, but in reading this I often feel confused when a recursive parameter is not last in the parameter list of a function definition. This might be my subjective bias, so if you don't think this adds any value I can undo.